packages/eslint-config-sheriff/package.json

```diff
@@ -46,8 +46,9 @@
   "scripts": {
     "preinstall": "npx only-allow pnpm",
     "clean": "rm -rf .turbo dist",
-    "build": "tsup",
-    "build-watch": "tsup",
+    "re-bundle-types": "dts-bundle-generator -out-file=dist/index.d.ts ./dist/index.d.ts",
```

Comment:
Is this needed, or does dts.resolve resolve it? (If not, I might try experimentalDts, which should also work)